examples/experimental/noise-js/package.json

```diff
@@ -9,9 +9,9 @@
     "start": "webpack-dev-server"
   },
   "dependencies": {
-    "@waku/sdk": "0.0.18",
-    "@waku/noise": "0.0.3-31510da",
```

Comment:
please, check which one is latest version of noise

Reply:
updated to use current next tag